rust/src/lib/nm/query_apply/ip.rs

```diff
@@ -93,7 +93,14 @@ pub(crate) fn nm_ip_setting_to_nmstate6(
             | NmSettingIpMethod::Shared => (true, Some(false), Some(false)),
             NmSettingIpMethod::Auto => (true, Some(true), Some(true)),
             NmSettingIpMethod::Dhcp => (true, Some(true), Some(false)),
-            NmSettingIpMethod::Ignore => (true, Some(false), Some(false)),
+            NmSettingIpMethod::Ignore => {
+                // For ipv6.method = ignore, we should trust nispor to
+                // set correct IPv6 status
+                return InterfaceIpv6 {
+                    enabled_defined: false,
+                    ..Default::default()
+                };
+            }
```

Comment:
Can we add an debug log line here? If we find an issue related to this in the future it might be clear. Something like `Found NmSettingIpMethod::Ignore, rely on Nispor query` or something like that.

Reply:
Sure.